### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Krithisree**
--> Tasks for deactivating users
--> Export the user list to csv file on user index page

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -16,7 +19,10 @@ def index
     respond_to do |format|
       format.html { @users = [] }
       format.json { render "index" }
-    end
+      format.xml { render xml:  @users }
```
Comment (SocGee) [inline app/controllers/admin/users_controller.rb:22]: Are we using XML? If not can we remove it?

File: app/models/user.rb [hunk 2]
```diff
@@ -26,7 +26,8 @@
 #  email_notification_type      :integer          default("full_html_email")
 #  isactive                     :boolean          default(TRUE)
 #  last_activated_at            :datetime
-
+require 'open-uri'
+require 'elasticsearch/model'
```
Comment (SocGee) [inline app/models/user.rb:30]: can we remove these requires if it is not used?

File: app/controllers/admin/users_controller.rb (hunk 1 above)
Comment (Krithisree) [reply app/controllers/admin/users_controller.rb]: Removed it

File: app/models/user.rb (hunk 2 above)
Comment (Krithisree) [reply app/models/user.rb]: open-uri is used by csv, removed elasticsearch

File: app/controllers/admin/users_controller.rb (hunk 1 above)
Comment (SocGee) [reply app/controllers/admin/users_controller.rb]: thanks, looks good

File: app/models/user.rb (hunk 2 above)
Comment (SocGee) [reply app/models/user.rb]: thanks, looks good

**Pull request merged**